### PR TITLE
[AMORO-2118] Dynamic chart in transactions page

### LINKED ISSUE #2118: [Subtask]: Dynamic chart in 'transactions' page
### Search before asking

- [X] I have searched in the [issues](https://github.com/NetEase/amoro/issues?q=is%3Aissue) and found no similar issues.

### Description

After integrating Paimon, the line chart on the Transactions page needs to display different content based on parameters from the backend, so the data structure of the line chart was separately extracted in #2116 named 'recordsSummaryForChart' and ‘filesSummaryForChart’. Like：
```
  long transactionId; // required
  int fileCount; // required
  long fileSize; // required
  long commitTime; // required
  String operation; // required
  Map<String, String> summary;

  //Determine what content to display on the front-end line chart.
  Map<String, String> recordsSummaryForChart;

  Map<String, String> filesSummaryForChart;
```

The front-end needs to use the keys in 'records/filesSummaryForChart' as category names and the values in 'records/filesSummaryForChart' as the values.

### Parent issue

#1876 

### Are you willing to submit PR?

- [ ] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by minteliuwm**
## Why are the changes needed?

Close #2118.

## Brief change log

- Dynamic chart in transactions page

## How was this patch tested?

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

- [x] Run test locally before making a pull request

## Documentation

- Does this pull request introduce a new feature? (no)
- If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)

### CODE REVIEW

**Comment by shidayang** [comment]
Please push complied files

**Comment by shidayang** [comment]
image

File: ams/dashboard/src/views/tables/components/Transactions.vue [hunk 1]
```diff
@@ -123,10 +123,18 @@ async function getTableInfo() {
     const fcData: ILineChartOriginalData = {}
     list.forEach((p: TransactionItem) => {
       // Assume that the time will not conflict and use the time as the unique key without formatting it.
-      const { summary } = p
+      const { summary, recordsSummaryForChart, filesSummaryForChart, commitTime } = p
       const { 'total-records': totalRecords, 'total-equality-deletes': totalEqualityDeletes, 'total-position-deletes': totalPositionDeletes, 'total-data-files': totalDataFiles, 'total-delete-files': totalDeleteFiles } = summary
```
Comment (wangtaohz) [inline ams/dashboard/src/views/tables/components/Transactions.vue:127]: If I understand correctly, `summary` should no longer be in use for charting, so we can simplify the code, and only support getting data from `recordsSummaryForChart` and `filesSummaryForChart` for charting. 😄 @minteliuwm

**Review by baiyangtx — APPROVED** [review]
LGTM,

**Pull request merged**